From af468c3a8f5d2c5f6323571419e03d1a6fd9869a Mon Sep 17 00:00:00 2001
From: stanleyk <stanleyk@ae88bc3d-4319-0410-8dbf-d08b4c9d3795>
Date: Mon, 18 Feb 2013 01:59:32 +0000
Subject: [PATCH 3/7] Removed extraneous characters which were preventing
 compilation of SSL_Context.cpp

Backported from: svn://svn.dre.vanderbilt.edu/DOC/Middleware/trunk@96831
---
 ACE/ChangeLog               | 12 ++++++++++++
 ACE/ace/SSL/SSL_Context.cpp | 12 ++++++------
 2 files changed, 18 insertions(+), 6 deletions(-)

diff --git a/ACE/ChangeLog b/ACE/ChangeLog
index 4bc64ab..74a7148 100644
--- a/ACE/ChangeLog
+++ b/ACE/ChangeLog
@@ -1,3 +1,15 @@
+Mon Feb 18 01:57:31 UTC 2013  Kevin Stanley  <stanleyk@ociweb.com>
+
+        * ace/SSL/SSL_Context.cpp:
+
+          Removed what appeared to be extraneous characters '3D'
+          from cut and paste.
+
+Sun Feb 17 03:37:42 UTC 2013  Douglas C. Schmidt  <schmidt@dre.vanderbilt.edu>
+
+        * ace/SSL/SSL_Context.h (ACE_SSL_Context): Added a missing comma.
+          Thanks to Johnny for reporting this.
+
 Sat Feb 16 21:53:26 UTC 2013  Kevin Stanley  <stanleyk@ociweb.com>
 
         * ace/SSL/SSL_Context.h:
diff --git a/ACE/ace/SSL/SSL_Context.cpp b/ACE/ace/SSL/SSL_Context.cpp
index 906bac6..34bc033 100644
--- a/ACE/ace/SSL/SSL_Context.cpp
+++ b/ACE/ace/SSL/SSL_Context.cpp
@@ -282,24 +282,24 @@ ACE_SSL_Context::set_mode (int mode)
       break;
 #ifdef TLS1_1_VERSION
     case ACE_SSL_Context::TLSv1_1_client:
-      method =3D ::TLSv1_1_client_method ();
+      method = ::TLSv1_1_client_method ();
       break;
     case ACE_SSL_Context::TLSv1_1_server:
-      method =3D ::TLSv1_1_server_method ();
+      method = ::TLSv1_1_server_method ();
       break;
     case ACE_SSL_Context::TLSv1_1:
-      method =3D ::TLSv1_1_method ();
+      method = ::TLSv1_1_method ();
       break;
 #endif
 #ifdef TLS1_2_VERSION
     case ACE_SSL_Context::TLSv1_2_client:
-      method =3D ::TLSv1_2_client_method ();
+      method = ::TLSv1_2_client_method ();
       break;
     case ACE_SSL_Context::TLSv1_2_server:
-      method =3D ::TLSv1_2_server_method ();
+      method = ::TLSv1_2_server_method ();
       break;
     case ACE_SSL_Context::TLSv1_2:
-      method =3D ::TLSv1_2_method ();
+      method = ::TLSv1_2_method ();
       break;
 #endif
     default:
-- 
1.7.12.4

